### DISCUSSION TITLE
How to rename configurations

### DISCUSSION THREAD

**Discussion by kec2** [BlitterStudio/amiberry#801] [Q&A] [answered]
I have made a configurations from a floppy disk. Lets say that the game consists of two Lemmings_Disk1.adf and Lemmings_Disk2.adf.
When I from Floppy drives menu clicks the "Save config for disk" when i get a configuration named "Lemmings_Disk1". 
Now how do I rename this configurations to "Lemmings"? Or better all configurations are save without the _Disk#.

**Comment by midwan** [comment, marked as answer]
Configurations are just files, so you could rename them like any other file.
However, they are meant to have the same name as the game, in order to be detected and loaded automatically. If you change the config name, but not the game name, then it will not be automatically loaded (if that's what you want, then it's OK).

**Reply by kec2** [reply]
I know that I can rename the files in a terminal but I would like do that within Amiberry.
I was now aware of the connection between the game file name a configuration file name.
What a would like was that the list of configurations did not all end with _Disk1. If everything is postfixed with something then that something looses is meaning.

**Reply by midwan** [reply]
I understand what you mean, but I'm not sure we can fix that. It was designed to look for config files with the same filename as the loaded "disk", to find a match.
One alternative would be to use WHDLoad `.lha` files instead of floppy images, since then you'd have a single file for the game, and a single config with the same name (which would not mention "_DiskX" at the end).

**Reply by kec2** [reply]
That's fine. I'll look into WHDLoad.

**Reply by amyren1966** [reply]
You can rename any config from amiberry in the configuration menu. Select the config and Load it, then change the name to what you want and press Save. This will create a duplicate of the config, with the new name. 
Then select the old config and press the delete button.
